bundle/Resources/config/view.yml

```diff
@@ -45,7 +42,8 @@ services:
             - '@ezpublish.api.repository'
             - 'eZ\Publish\Core\MVC\Symfony\Matcher\ContentBased'
             - '@ezpublish.config.resolver'
-            - 'content_view_defaults'
+            - 'ngcontent_view_defaults'
         calls:
             - [setContainer, ["@service_container"]]
+
```

Comment:
The blank line here looks like a mistake?

Reply:
Indeed, fixed in https://github.com/netgen/ezplatform-site-api/pull/130/commits/d8692fb1d11e2f3c2e18644bd0f3f9e5cbb27784